layout/theme.liquid

```diff
@@ -51,6 +51,46 @@
       {{ body_font_bold_italic | font_face: font_display: 'swap' }}
       {{ settings.type_header_font | font_face: font_display: 'swap' }}
 
+      {% for scheme in settings.color_schemes %}
+        {% if forloop.index == 1 %}
+          :root,
```

Comment:
Effectively. This makes "scheme 1" the fallback scheme for areas that don't have a scheme setting or a scheme parent to inherit from, like the site background. Though it’s not as much a “default” as it is a “fallback”. Our sections mostly all default to background-1, which works nicely when the default and the fallback schemes are the same. If our fallback isn't background-1, the site background and the sections (in their initial state) will be different colors.

So.. an alternative is to specifically check for `background-1` rather than index == 1 to match the default scheme we use in most of our sections, but my current thinking is we shouldn't commit to specific scheme ids existing. Figured I'd call that option out though in case we want to challenge that.